docs/src/pages/docs/integrations/css.md

```diff
@@ -76,12 +76,14 @@ export default {
       embedFiles: false,
       /** (optional) transform specific token values */
       transform: () => null,
-      /** (optional) add custom namespace to CSS vars */
+      /** (deprecated, use generateName instead) add custom namespace to CSS vars */
```

Comment:
Thanks for updating the JSDoc hints! I can make this change after merge, but I’ll put a `/** @deprecated` tag on it and that shows up in the IDE